android/app/src/main/res/values-ca/strings.xml

```diff
@@ -38,7 +38,8 @@
 	<string name="about_menu_title">Quant a l\'Organic Maps</string>
 	<!-- Text in About screen -->
 	<string name="about_proposition_2">• Estalvia la bateria, no utilitza dades de fons</string>
-	<string name="connection_settings">Paràmetres de connexió</string>
+	<!-- The button that opens system location settings -->
+	<string name="location_settings">Configuració d\'ubicació</string>
```

Comment:
I'd maintain "paràmetres" as a translation for "settings"; it's long been the preferred term of the Softcatalà organization.